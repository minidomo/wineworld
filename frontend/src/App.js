import React from 'react';
import './App.css';
import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';

import Navbar from './components/NavBar';
import Home from './containers/Home';
import About from './containers/About';


import Wines from './containers/WineModel/Wines';
import Wine1 from './containers/WineModel/Wine1';
import Wine2 from './containers/WineModel/Wine2';
import Wine3 from './containers/WineModel/Wine3';

import Vineyards from './containers/VineyardModel/Vineyards';
import Vineyard1 from './containers/VineyardModel/Vineyard1';
import Vineyard2 from './containers/VineyardModel/Vineyard2';
import Vineyard3 from './containers/VineyardModel/Vineyard3';


import Regions from './containers/RegionModel/Regions';
import Region1 from './containers/RegionModel/Region1';
import Region2 from './containers/RegionModel/Region2';
import Region3 from './containers/RegionModel/Region3';


const App = () => {
  return (
    <Router>
      <div className="App">
        <Navbar />
        <div className="Content">
          <Routes>
            <Route path="/" element={<Home />}></Route>


            <Route path="/Wines" element={<Wines />}></Route>
            <Route path="/Wines/Wine1" element={<Wine1 />}></Route>
            <Route path="/Wines/Wine2" element={<Wine2 />}></Route>
            <Route path="/Wines/Wine3" element={<Wine3 />}></Route>


            <Route path="/Vineyards" element={<Vineyards />}></Route>
            <Route path="/Vineyards/Vineyard1" element={<Vineyard1 />}></Route>
            <Route path="/Vineyards/Vineyard2" element={<Vineyard2 />}></Route>
            <Route path="/Vineyards/Vineyard3" element={<Vineyard3 />}></Route>


            <Route exact path="/Regions" element={<Regions />}></Route>
            <Route path="/Regions/Region1" element={<Region1 />}></Route>
            <Route path="/Regions/Region2" element={<Region2 />}></Route>
            <Route path="/Regions/Region3" element={<Region3 />}></Route>


            <Route path="/About" element={<About />}></Route>
          </Routes>
        </div>
        
      </div>
    </Router>
  );
}

export default App;



