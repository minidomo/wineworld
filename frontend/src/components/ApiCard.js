import React from 'react';
import Button from 'react-bootstrap/Button';
import Card from 'react-bootstrap/Card';
import { Link } from 'react-router-dom';


const ApiCard = props => {
  const { name, image, text, url } = props.api;

  return (
    <Card style={{ width: '18rem' }}>
      <Card.Img variant="top" src={image} />
      <Card.Body>
        <Card.Title>{name}</Card.Title>
        <Card.Text>{text}</Card.Text>
        <Link to={url} class={'btn stretched-link'}>
        </Link>
      </Card.Body>
    </Card>
  );
};

export default ApiCard;
