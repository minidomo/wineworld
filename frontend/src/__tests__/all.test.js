import renderer from "react-test-renderer";
import RayImage from "../assets/rayPic.jpg";
import WineCard from "../components/WineCard";
import RegionCard from "../components/RegionCard";
import VineyardCard from "../components/VineyardCard";
import DeveloperCard from "../components/DeveloperCard";
import HomeCard from "../components/HomeCard";
import ToolCard from "../components/ToolCard";
import GitLabImage from "../assets/gitlab.png";
import ApiCard from "../components/ApiCard";
import WineImage from "../assets/wine.jpg";
import ReactImage from "../assets/logo512.png";
import NavBar from "../components/NavBar";
import { BrowserRouter } from "react-router-dom";
import App from "../App.js";
import About from "../containers/About.js";

it("Init Wine", () => {
  const wine = {
    "country": "United States",
    "id": 310,
    "image": "https://images.vivino.com/thumbs/_IiGnDHSSaOc9u9c9CzVSA_pb_x300.png",
    "name": "Beckstoffer Dr. Crane Vineyard 2012",
    "rating": 4.9,
    "region": "St. Helena",
    "reviews": 32,
    "type": "Red",
    "winery": "Realm"
  };
  const component = renderer.create(<BrowserRouter><WineCard wine={wine} /></BrowserRouter>);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

it("Init Vineyard", () => {
  const vineyard = {
    "country": "United States",
    "id": 202,
    "image": "https://s3-media4.fl.yelpcdn.com/bphoto/Sam2Wv5bvNpEtcTLNYeB3g/o.jpg",
    "name": "Carhartt Family Wines",
    "price": 2,
    "rating": 4.5,
    "reviews": 561,
    "url": "https://www.yelp.com/biz/carhartt-family-wines-los-olivos"
  };
  const component = renderer.create(<BrowserRouter><VineyardCard vineyard={vineyard} /></BrowserRouter>);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

it("Init Region", () => {
  const region = {
    "country": "United States",
    "id": 3,
    "image": {
      "height": 450,
      "url": "https://media-cdn.tripadvisor.com/media/photo-s/19/8b/fb/16/die-neverland-ranch-war.jpg",
      "width": 450
    },
    "name": "Sta. Rita Hills",
    "rating": 4.1,
    "related": {
      "vineyards": [
        {
          "country": "United States",
          "id": 202,
          "image": "https://s3-media4.fl.yelpcdn.com/bphoto/Sam2Wv5bvNpEtcTLNYeB3g/o.jpg",
          "name": "Carhartt Family Wines",
          "price": 2,
          "rating": 4.5,
          "reviews": 561,
          "url": "https://www.yelp.com/biz/carhartt-family-wines-los-olivos"
        },
        {
          "country": "United States",
          "id": 203,
          "image": "https://s3-media2.fl.yelpcdn.com/bphoto/wHilkiD9MUr1CIJMutiMow/o.jpg",
          "name": "Saarloos & Sons",
          "price": 2,
          "rating": 4.5,
          "reviews": 507,
          "url": "https://www.yelp.com/biz/saarloos-and-sons-los-olivos"
        },
        {
          "country": "United States",
          "id": 204,
          "image": "https://s3-media1.fl.yelpcdn.com/bphoto/TpPsvmQVES-F47GB31hCNA/o.jpg",
          "name": "Dreamc√¥te Wine Co.",
          "price": 2,
          "rating": 4.5,
          "reviews": 154,
          "url": "https://www.yelp.com/biz/dreamc%C3%B4te-wine-co-los-olivos"
        }
      ],
      "wines": [
        {
          "country": "United States",
          "id": 17,
          "image": "https://images.vivino.com/thumbs/DtBMhsMBTtiA4URvVaUBNA_pb_x300.png",
          "name": "Rattrapante Grenache 2012",
          "rating": 4.9,
          "region": "Sta. Rita Hills",
          "reviews": 34,
          "type": "Red",
          "winery": "Sine Qua Non"
        },
        {
          "country": "United States",
          "id": 18,
          "image": "https://images.vivino.com/thumbs/hhlGZQVnR7C_RYEn8e677g_pb_x300.png",
          "name": "Chardonnay 2016",
          "rating": 4.6,
          "region": "Sta. Rita Hills",
          "reviews": 38,
          "type": "White",
          "winery": "Sea Smoke"
        },
        {
          "country": "United States",
          "id": 364,
          "image": "https://images.vivino.com/thumbs/czzcC4ZaStGChaDkmFYmww_pb_x300.png",
          "name": "The 17th Nail in My Cranium 2005",
          "rating": 4.8,
          "region": "Sta. Rita Hills",
          "reviews": 58,
          "type": "Red",
          "winery": "Sine Qua Non"
        },
        {
          "country": "United States",
          "id": 376,
          "image": "https://images.vivino.com/thumbs/eoESQg64S8mw1BwEJVVkGQ_pb_x300.png",
          "name": "Stockholm Syndrome Eleven Confession Vineyard Grenache N.V.",
          "rating": 4.8,
          "region": "Sta. Rita Hills",
          "reviews": 42,
          "type": "Red",
          "winery": "Sine Qua Non"
        },
        {
          "country": "United States",
          "id": 379,
          "image": "https://images.vivino.com/thumbs/eoESQg64S8mw1BwEJVVkGQ_pb_x300.png",
          "name": "Stockholm Syndrome Eleven Confession Vineyard Grenache 2010",
          "rating": 4.8,
          "region": "Sta. Rita Hills",
          "reviews": 40,
          "type": "Red",
          "winery": "Sine Qua Non"
        }
      ]
    },
    "reviews": 90,
    "tags": [
      "Shopping",
      "Wine Bars",
      "Activities",
      "Wineries & Vineyards",
      "Food & Drink",
      "Gift & Specialty Shops",
      "Nightlife"
    ],
    "tripTypes": [
      "Couples",
      "Friends getaway"
    ],
    "url": "https://www.tripadvisor.com/Tourism-g32660-Los_Olivos_California-Vacations.html",
  };
  const component = renderer.create(<BrowserRouter><RegionCard region={region} /></BrowserRouter>);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

it("Init Developer", () => {
  const Developer = {
    name: "Ray Yin",
    gitlab_id: "jrayyin",
    role: "Front-end",
    bio: " I'm a junior studying Computer Science at UT Austin. In my free time I enjoy playing volleyball and trying new restaurants.",
    commits: 0,
    issues: 0,
    unit_tests: 10,
    image: RayImage,
  };
  const component = renderer.create(<DeveloperCard devCard={Developer} />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

it("Init Api", () => {
  const Api = {
    name: "GitLab API",
    image: GitLabImage,
    text: "A Javascript library for front-end development",
    url: "https://docs.gitlab.com/ee/api/",
  };
  const component = renderer.create(<ApiCard api={Api} />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

it("Init Tool", () => {
  const Tool = {
    name: "React",
    image: ReactImage,
    text: "A Javascript library for front-end development",
    url: "https://reactjs.org/",
  };
  const component = renderer.create(<ToolCard tool={Tool} />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

it("Init NavBar", () => {
  const component = renderer.create(<BrowserRouter><NavBar /></BrowserRouter>);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

it("Init App", () => {
  const component = renderer.create(<App />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

it("Init About", () => {
  const component = renderer.create(<About />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});


it("Init HomeCard", () => {
  const Home = {
    title: "Wine",
    image: WineImage,
    text: "Check out some wines!",
    link: "https://www.google.com/",
  };
  const component = renderer.create(<BrowserRouter><HomeCard home={Home} /></BrowserRouter>);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});


