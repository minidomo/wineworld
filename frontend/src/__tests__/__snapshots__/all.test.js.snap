// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Init About 1`] = `
<div
  className="vstack"
>
  <div
    className="p-4 container"
  />
  <div
    className="p-4 container"
    style={
      Object {
        "backgroundColor": "#BBB",
      }
    }
  >
    <h1
      className="d-flex justify-content-center p-4 "
    >
      Welcome to WineWorld!
    </h1>
    <p
      className="mx-auto"
    >
      WineWorld is an innovative platform that provides a comprehensive guide to the world of wine, vineyards, and wine regions. Our goal is to empower wine enthusiasts, from beginners to experts, to explore and deepen their understanding of this fascinating industry.
    </p>
    <h1
      className="d-flex justify-content-center p-3 "
    >
      Repository Statistics
    </h1>
    <div
      className="p-4 row"
    >
      <div
        className="d-flex justify-content-center col"
      >
        <h2>
          Total Commits: 
          0
        </h2>
      </div>
      <div
        className="d-flex justify-content-center col"
      >
        <h2>
          Total Issues: 
          0
        </h2>
      </div>
      <div
        className="d-flex justify-content-center col"
      >
        <h2>
          Total Tests: 
          0
        </h2>
      </div>
    </div>
    <a
      href="https://documenter.getpostman.com/view/21507814/2s93CEvGRv"
    >
      <h2>
        API Documentation
      </h2>
    </a>
  </div>
  <div
    className="p-4 container"
  >
    <h1
      className="d-flex justify-content-center p-4 "
    >
      Meet the WineWorld Team!
    </h1>
    <div
      className="row"
    >
      <div
        className="d-flex justify-content-center col"
      >
        <div
          className="spinner-grow"
        />
      </div>
    </div>
  </div>
  <div
    className="p-4 container"
  >
    <h1
      className="d-flex justify-content-center p-4 "
    >
      Tools
    </h1>
    <div
      className="g-4 p-4 justify-content-center row row-cols-xl-5 row-cols-md-3 row-cols-sm-2 row-cols-1"
    >
      <div
        className="d-flex align-self-stretch col"
      >
        <div
          className="card"
          style={
            Object {
              "width": "18rem",
            }
          }
        >
          <img
            className="card-img-top"
            src="logo512.png"
          />
          <div
            className="card-body"
          >
            <div
              className="card-title h5"
            >
              React
            </div>
            <p
              className="card-text"
            >
              A Javascript library for front-end development
            </p>
            <a
              href="https://reactjs.org/"
            >
              <button
                className="btn btn-primary"
                disabled={false}
                type="button"
              >
                Learn More
              </button>
            </a>
          </div>
        </div>
      </div>
      <div
        className="d-flex align-self-stretch col"
      >
        <div
          className="card"
          style={
            Object {
              "width": "18rem",
            }
          }
        >
          <img
            className="card-img-top"
            src="gitlab.png"
          />
          <div
            className="card-body"
          >
            <div
              className="card-title h5"
            >
              GitLab
            </div>
            <p
              className="card-text"
            >
              Collaborative software development platform
            </p>
            <a
              href="https://about.gitlab.com/"
            >
              <button
                className="btn btn-primary"
                disabled={false}
                type="button"
              >
                Learn More
              </button>
            </a>
          </div>
        </div>
      </div>
      <div
        className="d-flex align-self-stretch col"
      >
        <div
          className="card"
          style={
            Object {
              "width": "18rem",
            }
          }
        >
          <img
            className="card-img-top"
            src="mteams.png"
          />
          <div
            className="card-body"
          >
            <div
              className="card-title h5"
            >
              Microsoft Teams
            </div>
            <p
              className="card-text"
            >
              Designated communication platform
            </p>
            <a
              href="https://www.microsoft.com/en-us/microsoft-teams/group-chat-software"
            >
              <button
                className="btn btn-primary"
                disabled={false}
                type="button"
              >
                Learn More
              </button>
            </a>
          </div>
        </div>
      </div>
      <div
        className="d-flex align-self-stretch col"
      >
        <div
          className="card"
          style={
            Object {
              "width": "18rem",
            }
          }
        >
          <img
            className="card-img-top"
            src="bootstrap.png"
          />
          <div
            className="card-body"
          >
            <div
              className="card-title h5"
            >
              React Bootstrap
            </div>
            <p
              className="card-text"
            >
              Bootstrap framework for React
            </p>
            <a
              href="https://react-bootstrap.github.io/"
            >
              <button
                className="btn btn-primary"
                disabled={false}
                type="button"
              >
                Learn More
              </button>
            </a>
          </div>
        </div>
      </div>
      <div
        className="d-flex align-self-stretch col"
      >
        <div
          className="card"
          style={
            Object {
              "width": "18rem",
            }
          }
        >
          <img
            className="card-img-top"
            src="postman.png"
          />
          <div
            className="card-body"
          >
            <div
              className="card-title h5"
            >
              Postman
            </div>
            <p
              className="card-text"
            >
              Platform for API development and testing
            </p>
            <a
              href="https://www.postman.com/"
            >
              <button
                className="btn btn-primary"
                disabled={false}
                type="button"
              >
                Learn More
              </button>
            </a>
          </div>
        </div>
      </div>
      <div
        className="d-flex align-self-stretch col"
      >
        <div
          className="card"
          style={
            Object {
              "width": "18rem",
            }
          }
        >
          <img
            className="card-img-top"
            src="aws.png"
          />
          <div
            className="card-body"
          >
            <div
              className="card-title h5"
            >
              AWS Amplify
            </div>
            <p
              className="card-text"
            >
              Hosting platform for WineWorld
            </p>
            <a
              href="https://aws.amazon.com/amplify/"
            >
              <button
                className="btn btn-primary"
                disabled={false}
                type="button"
              >
                Learn More
              </button>
            </a>
          </div>
        </div>
      </div>
      <div
        className="d-flex align-self-stretch col"
      >
        <div
          className="card"
          style={
            Object {
              "width": "18rem",
            }
          }
        >
          <img
            className="card-img-top"
            src="namecheap.png"
          />
          <div
            className="card-body"
          >
            <div
              className="card-title h5"
            >
              Namecheap
            </div>
            <p
              className="card-text"
            >
              Domain name registration for site
            </p>
            <a
              href=""
            >
              <button
                className="btn btn-primary"
                disabled={false}
                type="button"
              >
                Learn More
              </button>
            </a>
          </div>
        </div>
      </div>
      <div
        className="d-flex align-self-stretch col"
      >
        <div
          className="card"
          style={
            Object {
              "width": "18rem",
            }
          }
        >
          <img
            className="card-img-top"
            src="vscode.png"
          />
          <div
            className="card-body"
          >
            <div
              className="card-title h5"
            >
              Visual Studio Code
            </div>
            <p
              className="card-text"
            >
              Popular code editing software
            </p>
            <a
              href="https://code.visualstudio.com/"
            >
              <button
                className="btn btn-primary"
                disabled={false}
                type="button"
              >
                Learn More
              </button>
            </a>
          </div>
        </div>
      </div>
      <div
        className="d-flex align-self-stretch col"
      >
        <div
          className="card"
          style={
            Object {
              "width": "18rem",
            }
          }
        >
          <img
            className="card-img-top"
            src="npm.png"
          />
          <div
            className="card-body"
          >
            <div
              className="card-title h5"
            >
              NPM
            </div>
            <p
              className="card-text"
            >
              Javascript package manager
            </p>
            <a
              href="https://www.npmjs.com/"
            >
              <button
                className="btn btn-primary"
                disabled={false}
                type="button"
              >
                Learn More
              </button>
            </a>
          </div>
        </div>
      </div>
      <div
        className="d-flex align-self-stretch col"
      >
        <div
          className="card"
          style={
            Object {
              "width": "18rem",
            }
          }
        >
          <img
            className="card-img-top"
            src="route53.png"
          />
          <div
            className="card-body"
          >
            <div
              className="card-title h5"
            >
              Route 53 AWS
            </div>
            <p
              className="card-text"
            >
              Routing from amplify to domain
            </p>
            <a
              href="https://aws.amazon.com/route53/"
            >
              <button
                className="btn btn-primary"
                disabled={false}
                type="button"
              >
                Learn More
              </button>
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="p-4 container"
  >
    <h1
      className="d-flex justify-content-center p-4 "
    >
      APIs
    </h1>
    <div
      className="g-4 p-4 justify-content-center row row-cols-xl-5 row-cols-md-3 row-cols-sm-2 row-cols-1"
    >
      <div
        className="d-flex align-self-stretch col"
      >
        <div
          className="card"
          style={
            Object {
              "width": "18rem",
            }
          }
        >
          <img
            className="card-img-top"
            src="gitlab.png"
          />
          <div
            className="card-body"
          >
            <div
              className="card-title h5"
            >
              GitLab API
            </div>
            <p
              className="card-text"
            >
              A Javascript library for front-end development
            </p>
            <a
              href="https://docs.gitlab.com/ee/api/"
            >
              <button
                className="btn btn-primary"
                disabled={false}
                type="button"
              >
                Learn More
              </button>
            </a>
          </div>
        </div>
      </div>
      <div
        className="d-flex align-self-stretch col"
      >
        <div
          className="card"
          style={
            Object {
              "width": "18rem",
            }
          }
        >
          <img
            className="card-img-top"
            src="wine.jpg"
          />
          <div
            className="card-body"
          >
            <div
              className="card-title h5"
            >
              Wines API
            </div>
            <p
              className="card-text"
            >
              A Javascript library for front-end development
            </p>
            <a
              href="https://sampleapis.com/api-list/wines"
            >
              <button
                className="btn btn-primary"
                disabled={false}
                type="button"
              >
                Learn More
              </button>
            </a>
          </div>
        </div>
      </div>
      <div
        className="d-flex align-self-stretch col"
      >
        <div
          className="card"
          style={
            Object {
              "width": "18rem",
            }
          }
        >
          <img
            className="card-img-top"
            src="yelp.jpg"
          />
          <div
            className="card-body"
          >
            <div
              className="card-title h5"
            >
              Yelp Fusion API
            </div>
            <p
              className="card-text"
            >
              Collaborative software development platform
            </p>
            <a
              href="https://fusion.yelp.com/"
            >
              <button
                className="btn btn-primary"
                disabled={false}
                type="button"
              >
                Learn More
              </button>
            </a>
          </div>
        </div>
      </div>
      <div
        className="d-flex align-self-stretch col"
      >
        <div
          className="card"
          style={
            Object {
              "width": "18rem",
            }
          }
        >
          <img
            className="card-img-top"
            src="tripadvisor.png"
          />
          <div
            className="card-body"
          >
            <div
              className="card-title h5"
            >
              Tripadvisor API
            </div>
            <p
              className="card-text"
            >
              Designated communication platform
            </p>
            <a
              href="https://www.tripadvisor.com/developers"
            >
              <button
                className="btn btn-primary"
                disabled={false}
                type="button"
              >
                Learn More
              </button>
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Init Api 1`] = `
<div
  className="card"
  style={
    Object {
      "width": "18rem",
    }
  }
>
  <img
    className="card-img-top"
    src="gitlab.png"
  />
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    >
      GitLab API
    </div>
    <p
      className="card-text"
    >
      A Javascript library for front-end development
    </p>
    <a
      href="https://docs.gitlab.com/ee/api/"
    >
      <button
        className="btn btn-primary"
        disabled={false}
        type="button"
      >
        Learn More
      </button>
    </a>
  </div>
</div>
`;

exports[`Init App 1`] = `
<div
  className="App"
>
  <nav
    class="navbar bg-[object Object]"
    data-bs-theme={
      Object {
        "ColorTheme": [Function],
      }
    }
  >
    <nav
      class="navbar fixed-top navbar-expand-lg bg-body-tertiary"
    >
      <div
        class="container-fluid"
      >
        <a
          class="navbar-brand"
          href="/"
          onClick={[Function]}
        >
          WineWorld
        </a>
        <button
          aria-controls="navbarText"
          aria-expanded="false"
          aria-label="Toggle navigation"
          class="navbar-toggler"
          data-bs-target="#navbarText"
          data-bs-toggle="collapse"
          type="button"
        >
          <span
            class="navbar-toggler-icon"
          />
        </button>
        <div
          class="collapse navbar-collapse"
          id="navbarText"
        >
          <ul
            class="navbar-nav me-auto mb-2 mb-lg-0"
          >
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/about"
                onClick={[Function]}
              >
                About
              </a>
            </li>
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/wines"
                onClick={[Function]}
              >
                Wines
              </a>
            </li>
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/vineyards"
                onClick={[Function]}
              >
                Vineyards
              </a>
            </li>
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/regions"
                onClick={[Function]}
              >
                Regions
              </a>
            </li>
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/visualizations"
                onClick={[Function]}
              >
                Visualizations
              </a>
            </li>
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/provider"
                onClick={[Function]}
              >
                Provider Visualizations
              </a>
            </li>
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/"
                onClick={[Function]}
              >
                Light/Dark
              </a>
            </li>
          </ul>
        </div>
        <form
          className="d-flex"
          onSubmit={[Function]}
        >
          <input
            className="form-control"
            onChange={[Function]}
            placeholder="search"
            type="search"
          />
        </form>
      </div>
    </nav>
  </nav>
  <div
    className="Content"
  >
    <div>
      <img
        alt="..."
        class="img-fluid w-100"
        src="home_logo.jpg"
      />
      <p />
      <div
        class="container text-center"
      >
        <div
          class="row"
        >
          <div
            class="col"
          >
            <div
              class="card"
            >
              <img
                alt=""
                class="card-img-top"
                src="w_home.jpg"
              />
              <div
                class="card-body"
              >
                <h5
                  class="card-title"
                >
                  Wines
                </h5>
                <p
                  class="card-text"
                >
                  Find wines that satisfy your taste
                </p>
                <a
                  class="btn custom1 stretched-link"
                  href="/wines"
                  onClick={[Function]}
                >
                  Explore
                </a>
              </div>
            </div>
          </div>
          <div
            class="col"
          >
            <div
              class="card"
            >
              <img
                alt=""
                class="card-img-top"
                src="v_home.jpg"
              />
              <div
                class="card-body"
              >
                <h5
                  class="card-title"
                >
                  Vineyards
                </h5>
                <p
                  class="card-text"
                >
                  Discover amazing vineyards
                </p>
                <a
                  class="btn custom1 stretched-link"
                  href="/vineyards"
                  onClick={[Function]}
                >
                  Explore
                </a>
              </div>
            </div>
          </div>
          <div
            class="col"
          >
            <div
              class="card"
            >
              <img
                alt=""
                class="card-img-top"
                src="r_home.jpg"
              />
              <div
                class="card-body"
              >
                <h5
                  class="card-title"
                >
                  Regions
                </h5>
                <p
                  class="card-text"
                >
                  Explore new regions
                </p>
                <a
                  class="btn custom1 stretched-link"
                  href="/regions"
                  onClick={[Function]}
                >
                  Explore
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
      <p />
    </div>
  </div>
</div>
`;

exports[`Init Developer 1`] = `
<div
  className="card"
  style={
    Object {
      "height": "42rem",
      "width": "18rem",
    }
  }
>
  <img
    className="card-img-top"
    src="rayPic.jpg"
  />
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    >
      Ray Yin
    </div>
    <p
      className="card-text"
    >
       I'm a junior studying Computer Science at UT Austin. In my free time I enjoy playing volleyball and trying new restaurants.
    </p>
  </div>
  <div
    className="list-group-flush list-group"
    onKeyDown={[Function]}
  >
    <div
      className="list-group-item"
      data-rr-ui-event-key={null}
      onClick={[Function]}
    >
      Front-end
    </div>
    <div
      className="list-group-item"
      data-rr-ui-event-key={null}
      onClick={[Function]}
    >
       Commits: 
      0
    </div>
    <div
      className="list-group-item"
      data-rr-ui-event-key={null}
      onClick={[Function]}
    >
       Issues: 
      0
    </div>
    <div
      className="list-group-item"
      data-rr-ui-event-key={null}
      onClick={[Function]}
    >
       Unit Tests: 
      10
    </div>
  </div>
</div>
`;

exports[`Init HomeCard 1`] = `
<div
  class="card"
>
  <img
    alt=""
    class="card-img-top"
  />
  <div
    class="card-body"
  >
    <h5
      class="card-title"
    />
    <p
      class="card-text"
    />
    <a
      class="btn custom1 stretched-link"
      href="/"
      onClick={[Function]}
    >
      Explore
    </a>
  </div>
</div>
`;

exports[`Init NavBar 1`] = `
<nav
  class="navbar bg-[object Object]"
  data-bs-theme={
    Object {
      "ColorTheme": [Function],
    }
  }
>
  <nav
    class="navbar fixed-top navbar-expand-lg bg-body-tertiary"
  >
    <div
      class="container-fluid"
    >
      <a
        class="navbar-brand"
        href="/"
        onClick={[Function]}
      >
        WineWorld
      </a>
      <button
        aria-controls="navbarText"
        aria-expanded="false"
        aria-label="Toggle navigation"
        class="navbar-toggler"
        data-bs-target="#navbarText"
        data-bs-toggle="collapse"
        type="button"
      >
        <span
          class="navbar-toggler-icon"
        />
      </button>
      <div
        class="collapse navbar-collapse"
        id="navbarText"
      >
        <ul
          class="navbar-nav me-auto mb-2 mb-lg-0"
        >
          <li
            class="nav-item"
          >
            <a
              class="nav-link"
              href="/about"
              onClick={[Function]}
            >
              About
            </a>
          </li>
          <li
            class="nav-item"
          >
            <a
              class="nav-link"
              href="/wines"
              onClick={[Function]}
            >
              Wines
            </a>
          </li>
          <li
            class="nav-item"
          >
            <a
              class="nav-link"
              href="/vineyards"
              onClick={[Function]}
            >
              Vineyards
            </a>
          </li>
          <li
            class="nav-item"
          >
            <a
              class="nav-link"
              href="/regions"
              onClick={[Function]}
            >
              Regions
            </a>
          </li>
          <li
            class="nav-item"
          >
            <a
              class="nav-link"
              href="/visualizations"
              onClick={[Function]}
            >
              Visualizations
            </a>
          </li>
          <li
            class="nav-item"
          >
            <a
              class="nav-link"
              href="/provider"
              onClick={[Function]}
            >
              Provider Visualizations
            </a>
          </li>
          <li
            class="nav-item"
          >
            <a
              class="nav-link"
              href="/"
              onClick={[Function]}
            >
              Light/Dark
            </a>
          </li>
        </ul>
      </div>
      <form
        className="d-flex"
        onSubmit={[Function]}
      >
        <input
          className="form-control"
          onChange={[Function]}
          placeholder="search"
          type="search"
        />
      </form>
    </div>
  </nav>
</nav>
`;

exports[`Init Region 1`] = `
<div
  className="container"
>
  <div
    className="card"
    style={
      Object {
        "height": "30rem",
      }
    }
  >
    <img
      className="card-img-top"
      onError={[Function]}
      src="https://media-cdn.tripadvisor.com/media/photo-s/19/8b/fb/16/die-neverland-ranch-war.jpg"
      style={
        Object {
          "height": "40%",
          "objectFit": "cover",
          "width": "100%",
        }
      }
    />
    <div
      className="card-body"
    >
      <div
        className="card-title h5"
      >
         
        Sta. Rita Hills
         
      </div>
      <div
        className="card-subtitle h6"
      >
         
        United States
         
      </div>
      <p
        className="card-text"
      >
        <small>
          Rating: 
          4.1
          <br />
          Review Count: 
          90
          <br />
          Trip Type: 
          Couples, Friends getaway
        </small>
      </p>
    </div>
    <div
      class="card-footer"
    >
      <a
        class="btn custom1 stretched-link btn-sm"
        href="/regions/3"
        onClick={[Function]}
      >
        Explore Region
      </a>
    </div>
  </div>
</div>
`;

exports[`Init Tool 1`] = `
<div
  className="card"
  style={
    Object {
      "width": "18rem",
    }
  }
>
  <img
    className="card-img-top"
    src="logo512.png"
  />
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    >
      React
    </div>
    <p
      className="card-text"
    >
      A Javascript library for front-end development
    </p>
    <a
      href="https://reactjs.org/"
    >
      <button
        className="btn btn-primary"
        disabled={false}
        type="button"
      >
        Learn More
      </button>
    </a>
  </div>
</div>
`;

exports[`Init Vineyard 1`] = `
<div
  className="container"
>
  <div
    className="card"
    style={
      Object {
        "height": "30rem",
      }
    }
  >
    <img
      className="card-img-top"
      onError={[Function]}
      src="https://s3-media4.fl.yelpcdn.com/bphoto/Sam2Wv5bvNpEtcTLNYeB3g/o.jpg"
      style={
        Object {
          "height": "40%",
          "objectFit": "cover",
          "width": "100%",
        }
      }
    />
    <div
      className="card-body"
    >
      <div
        className="card-title h5"
      >
         
        Carhartt Family Wines
         
      </div>
      <div
        className="card-subtitle h6"
      >
         
        United States
         
      </div>
      <p
        className="card-text"
      >
        Price Level: 
        2
        <br />
        Rating: 
        4.5
        <br />
        Review Count: 
        561
      </p>
    </div>
    <div
      class="card-footer"
    >
      <a
        class="btn custom1 stretched-link btn-sm"
        href="/vineyards/202"
        onClick={[Function]}
      >
        See Vineyard
      </a>
    </div>
  </div>
</div>
`;

exports[`Init Wine 1`] = `
<div
  className="container"
>
  <div
    className="card"
    style={
      Object {
        "height": "35rem",
      }
    }
  >
    <img
      class="img-thumbnail"
      className="card-img-top"
      onError={[Function]}
      src="https://images.vivino.com/thumbs/_IiGnDHSSaOc9u9c9CzVSA_pb_x300.png"
      style={
        Object {
          "height": "40%",
          "objectFit": "contain",
          "width": "100%",
        }
      }
    />
    <div
      className="card-body"
    >
      <div
        className="card-title h5"
      >
         
        <small>
           
          Beckstoffer Dr. Crane Vineyard 2012
           
        </small>
         
      </div>
      <div
        className="card-subtitle h6"
      >
         
        Red
         Wine 
      </div>
      <p
        className="card-text"
      >
        Country: 
        United States
        <br />
        Region: 
        St. Helena
        <br />
        Winery: 
        Realm
        <br />
        Rating: 
        4.9
        <br />
        Reviews: 
        32
      </p>
    </div>
    <div
      class="card-footer"
    >
      <a
        class="btn custom1 stretched-link btn-sm"
        href="/wines/310"
        onClick={[Function]}
      >
        View Wine
      </a>
    </div>
  </div>
</div>
`;
